README.md

```diff
@@ -1 +1,301 @@
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!-General requirements
+Once you finish your project, double-check to make sure your repo is ready:
+
+Make sure that there are no linter errors.
+Make sure that you used correct (GitHub flow).
+Make sure that you documented your work.
+Requirements
+Remember to document your work in a professional way.
+Create a new GitHub repo.
+Create a new branch according to the GitHub flow rules.
+Add a .gitignore file.
+Add node_modules to gitignore file.
+Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
+Add descriptive README file to your project - please use this template but remember to customize it to your project.
+Add 3 files (each one in a separate commit):
+A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
+A .css file that includes one class with styles for your header (make it in your favorite color).
+A test.md file with the text "This file should be ignored by git".
+Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
+Open a pull request.-
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+git@github.com:CaptainBawa/Hello-Microverse.git
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] kindly personalize this part by adding your own social media link which will help the people visiting this project to easily reach you out or connect and also remember that you are only one Author